server/app/files.go

```diff
@@ -1,34 +1,91 @@
 package app
 
 import (
+	"errors"
 	"fmt"
 	"io"
 	"path/filepath"
 	"strings"
 
+	"github.com/mattermost/mattermost-server/v6/model"
```

Comment:
May want to alias this (even though not strictly necessary). May be easier to understand which model we're using.

Reply:
Makes sense, done!